src/main.zig

```diff
@@ -174,24 +182,43 @@ fn newll(key: Key, value: *const Slot, allocator: Allocator, crs: *CRS) !*LastLe
     return ll;
 }
 
+fn newemptyll(stem: Stem, depth: u8, allocator: Allocator, crs: *CRS) !*LastLevelNode {
+    var ll = try allocator.create(LastLevelNode);
+    ll.values = [_]?*Slot{null} ** 256;
+    ll.stem = [_]u8{0} ** 31;
+    ll.depth = depth;
+    std.mem.copy(u8, ll.stem[0..], stem[0..31]);
+    ll.crs = crs;
+    return ll;
+}
+
+const PoAStub = struct {
+    stem: *const Stem,
+    commitment: ?*Element,
+
+    fn computeCommitment(self: *const PoAStub) anyerror!Element {
+        return self.commitment.?.*;
+    }
+};
```

Comment:
I had to add this, for PoA stems. In go-verkle, we reuse a leaf, but I believe it's cleaner this way..